.github/workflows/coverage.yml

```diff
@@ -0,0 +1,40 @@
+on: [push]
+
+name: Code Coverage
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
+
+      - name: Clean
```

Comment:
Why do we need to clean?

Reply:
Docs recommended it, so I added just to be safe